_posts/blog/2018-10-11-review-of-periodicos-e-literatura.md

```diff
@@ -0,0 +1,27 @@
+
+The collection appears to consciously serve two communities: library staff and humanities researchers. The majority of the publications coming from the collection have been [*technical and research papers*]() in LIS journals. The Foundation is the biggest cultural heritage repository in the country and the NDLB shares this position as its digital parallel. All this to say that, the impact of the collection is significant considering its status as an example digital project development in Brazil and the fact that it has the potential to link itself with almost 2 million documents across thousands of other collections housed in the archives. There is also a productive tension between these communities and disciplines that can be summarized as an issue of "essences", as Ann Stoler proposes the term in (*Along the Archival Grain: Epistemic Anxieties and Colonial Commonsense* 4). 
+
+Both groups work for the mission of preserving Brazilian memory and patrimony. This is what could be termed the essence of the country. Nevertheless, in the LIS articles there is a trend of [*historicizing*](http://www.unirio.br/ppgb/arquivo/maria-ione-caser-da-costa/view) and [*challenging*](http://www.letras.ufrj.br/prisma/jornada4.htm) the [*evolution*](https://portal.febab.org.br/anais/article/view/1320) of the Foundation and its digital counterpart. Meanwhile, the humanities publications listed on the website don't focus on the digital objects being analyzed. This is also reflected in the monographs on the website where extensive archival work was done and is only reflected in the material description of the items and the bibliographic reference. There is an implied stability and fixity of meaning in this last method, whereas in the [*work by the librarians*](https://www.bn.gov.br/es/node/3709)](https://www.bn.gov.br/es/node/3709), the materials are evolving and shifting the national imaginary and the very conditions in which it was created. This is more in line with current interpretations of the "unique space" of intersecting traditions afforded by the analog and digital that Bonnie Mak calls attention to ("Archaeology of a Digitization" 1516). 
```

Comment:
This paragraph would benefit from a more clear and specific topic sentence. Shifting the final sentence of the previous paragraph down might be a solution, but it might need to be reframed as well.